keps/sig-scheduling/2249-pod-affinity-namespace-selector/README.md

```diff
@@ -129,11 +129,11 @@ Items marked with (R) are required *prior to targeting to a milestone / release*
 - [x] (R) Design details are appropriately documented
```

Comment:
The design detail here says that empty selector is not permitted, while the actual implementation was that empty matches all namespaces. Please update.

Reply:
the kep was updated with that change: https://github.com/kubernetes/enhancements/commit/b0462c479ab4ea7130f64a7a04f23841fdcf5830#diff-a7c0901341840621750e6d4a77dbddc75759e106ffdc3c3f7044cb12472ecfe2

but a comment was missed, updated.